src/test/java/com/github/dockerjava/core/command/CreateContainerCmdImplTest.java

```diff
@@ -45,7 +21,29 @@
 import java.util.Map;
 import java.util.UUID;
 
-import static org.hamcrest.MatcherAssert.assertThat;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
```

Comment:
What is the settings in your IDE about imports? 
Usually static imports are after generic imports and javax/java.lang are usually in separate blocks.